apps/validate/validate.py

```diff
@@ -149,6 +150,15 @@ def _validate_company_codes(self, *args):
         """Ignore company codes."""
         pass
 
+    def _validate_validate_characters(self, validate, field, value):
```

Comment:
can you add some test to see it in action? how does it know what field to check?

Reply:
sure, I will try to add some.
> how does it know what field to check?

In the content-profile added an option to enable and disable the character validation as we do for `required` validation.